web-console/src/utils/download.ts

```diff
@@ -107,3 +107,20 @@ export function downloadQueryResults(
   const lineBreak = '\n';
   downloadFile(lines.join(lineBreak), format, filename);
 }
+
+export function copyJSONResultsToClipboard(queryResult: QueryResult): void {
```

Comment:
This function appears to copy paste code from `downloadQueryResults`. I would prefer it if you extracted the functionality into a function called `queryResultToJsonlString(queryResult: QueryResult): string`  and then call it from both places!